lib/account_deleter.rb

```diff
@@ -56,6 +57,10 @@ def delete_standard_user_associations
     end
   end
 
+  def delete_user_invitation_code
+    InvitationCode.find_by(user_id: user.id).destroy
```

Comment:
With the comment below (at the test) taken into consideration, 

```suggestion
    InvitationCode.find_by(user_id: user.id).try(:destroy)
```

should make it unnecessary to add an invitation code to testing users, and it should make sure this always works for "real" users.